### PR TITLE
Fixed the passwords view

### PR DESCRIPTION
**Pull request by AlexanderKhapchenko**
I added a component to calm the QA 
I know there is an input field in the common folder
But it is a bit incomplete there
I decided to follow the example of the sign-in page and add my own modified input field 
Because if I fix it in the folder common, I think someone somewhere might break something

### CODE REVIEW

**Comment by AlexanderKhapchenko** [comment]
[[User setting layout] UI design does not meet mockup](https://trello.com/c/2NCWSsm1/82-user-setting-layout-ui-design-does-not-meet-mockup)

**Review by svitlua — CHANGES_REQUESTED** [review]
Please, dont make unnecessary copies of common components. You can refactor according the QA requirement and use FormInput component from src/components/common/form-input/
If we make copies of components it will be very hard to maitain the project in the future.
We also need to refactor usage of input field in SignIn Page to unify the look and functionality of form input. SignIn Page was created during the first days, and at that moment we didn't have opportunity to use common components. Lets refactor and not make thins more complicated.

**Comment by Paul-Vasylenko** [comment]
Agree with @svitlua , it would be great if we use the same component here. You can use unnecessary fields (e.g. {name?: string}), so that shouldn't crash anything ion other places. If the problem was with onClick for 'eye' you shoud just add that function to container and ,by props, send it to every component.

**As for other changes - looks great!**

**Review by Paul-Vasylenko — APPROVED** [review]

File: client/src/components/pages/setting/components/information/index.tsx [hunk 1]
```diff
@@ -2,18 +2,17 @@ import React from 'react';
...
 import List from '../list-radio';
 import { IFormItem, IInformationProps } from './interfaces';
 import styles from './information.module.scss';
 
 const SignupSchema = Yup.object().shape({
-	name: Yup.string().min(2, 'Too Short!').max(50, 'Too Long!').required('Required'),
-	username: Yup.string().min(2, 'Too Short!').max(50, 'Too Long!').required('Required'),
-	clan: Yup.string().min(2, 'Too Short!').max(50, 'Too Long!').required('Required'),
-	skills: Yup.string().min(2, 'Too Short!').max(50, 'Too Long!').required('Required'),
+	name: Yup.string().min(2, 'Too short').max(50, 'Too long').required('Required'),
```
Comment (svitlua) [inline client/src/components/pages/setting/components/information/index.tsx:12]: Please, use here QA requirements for validation:
https://trello.com/c/xusLg7a2/44-sign-in-sign-up-notifications

or simply copy from here:
https://github.com/BinaryStudioAcademy/bsa-2021-codetrainer/blob/patch/username/client/src/components/pages/sign-up/index.tsx

**Review by svitlua — APPROVED** [review]

**Pull request merged**